src/test/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessorTest.java

```diff
@@ -135,16 +144,19 @@ void GIVEN_one_system_component_one_file_less_than_max_WHEN_merge_THEN_reads_ent
             assertEquals(12, localDate.getMonth().getValue());
             assertEquals(17, localDate.getDayOfMonth());
         }
+        chan.close();
+        assertFalse(chan.isOpen());
```

Comment:
You are closing the channel on the previous line so this assertion is not really useful